docs/concepts/custom-itemspanel.md

```diff
@@ -0,0 +1,71 @@
+---
+description: CONCEPTS
+---
+
+import exampleScreenshot from '/img/concepts/customitemspanel/ItemsControl-with-ItemPanel-and-Styles.png';
+
+# Custom ItemsPanel
+
+the following built-in controls, display their items in a StackPanel by default: 
+- [`ItemsControl`](../reference/controls/itemscontrol.md)
+- [`TreeView`](../reference/controls/detailed-reference/treeview-1.md)
+- [`Carousel`](../reference/controls/carousel.md)
+- [`Menu`](../reference/controls/menu.md)
+- [`ComboBox`](../reference/controls/combobox.md)
+- `ListBox` 
+
+You can customise this default behaviour via the property `ItemsPanel` of the control. The example below demonstrate a customisation of an `ItemsControls`
```

Comment:
I would suggest moving this into some sort of introduction section and rewording it to something like:

> "All `ItemsControl`s have an item container panel which is used to layout their items. It is possible to override the type of panel used by the control to achieve custom/alternative layouts of items in a control. This document provides some examples showcasing how and why you would do this.

Reply:
Indeed, it's much better. You are clearly better with words than me.